RetailCoder.VBE/App.cs

```diff
@@ -71,17 +72,17 @@ public class App : IDisposable
             _parser.State.StatusMessageUpdate += State_StatusMessageUpdate;
             _stateBar.Refresh += _stateBar_Refresh;
 
-            var sink = new VBProjectsEventsSink();
+            _sink = new VBProjectsEventsSink();
```

Comment:
nice catch. I would like to see the COM event sinks moved to another `IDisposable` class responsible for it though. `App` has way too many responsibilities as it stands.